packages/SwingSet/src/liveslots/collectionManager.js

```diff
@@ -11,6 +11,7 @@ import {
   assertKeyPattern,
   assertPattern,
   matches,
+  fit,
```

Comment:
I intensely dislike the name `fit` for this function, because it reads as an imperative or a measurement rather than a test.  If you're pushing for terseness, `fits` would be better, though I'm still not wild about it.  I'm not sure this is a place where terseness has a lot of benefit.

Reply:
The possibility of renaming `fit` would be outside this PR anyway, right?